### PR TITLE
CommitBear: Fix regex for issue id

### PR DESCRIPTION
**Pull request by kriti21**
Closes https://github.com/coala/coala-bears/issues/2469

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by kriti21** [comment]
@jayvdb @damngamerz Please review.

**Comment by damngamerz** [comment]
LGTM :) Also nice work on improving those misleading texts :+1: 
@jayvdb This looks good.

**Comment by ksdme** [comment]
I think you should also solve https://github.com/coala/coala-bears/issues/2468 in the same PR.

**Comment by kriti21** [comment]
> 

> I think you should also solve #2468 in the same PR.

@jayvdb  has already made comments stating these are different issues and should be solved separately.

**Comment by jayvdb** [comment]
Your commit message says both fixes and closes.

Your shortlog needs to describe the change as accurately as possible.

File: setup.py [hunk 1]
```diff
@@ -119,7 +119,6 @@ def run(self):
               'Environment :: Plugins',
               'Environment :: MacOS X',
               'Environment :: Win32 (MS Windows)',
-              'Environment :: X11 Applications :: Gnome',
```
Comment (ksdme) [inline setup.py]: This change doesn't help the PR. If you think this is an valid change start a newcomer issue maybe?

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Ugh, gitmate rebase doesnt create a new commit if it is already rebased.

Please amend comment and force push to kick off new CI.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2484/commits/9332d3d1e63f6098ae4e39e53bbf8679dce4c9a3

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**